client/lib/posts/actions.js

```diff
@@ -157,6 +154,42 @@ PostActions = {
 	},
 
 	/**
+	 * Copy an existing post into a new draft and start keeping track of edits to it
+	 *
+	 * @param {Number} siteId Site ID to copy post from
+	 * @param {Number} postId Post ID to copy
+	 */
+	startEditingPostCopy( siteId, postId ) {
+		if ( ! siteId ) {
+			return;
+		}
+
+		wpcom.site( siteId ).post( postId ).get().then( post => {
+			this.startEditingNew( siteId, {
+				type: 'post',
+				content: post.content,
+			} );
+
+			const postAttributes = pick(
+				post,
+				'canonical_image',
+				'excerpt',
+				'featured_image',
+				'format',
+				'metadata',
+				'post_thumbnail',
+				'title'
```

Comment:
Still think it's odd that `startEditingNew` takes _some_ attributes, including `title`, while the rest of them have to be deferred to a subsequent edit. Had you considered updating `startEditingNew` to accept any post attributes from which the post should be initialized?